recipes/openssl/1.x.x/conanfile.py

```diff
@@ -699,7 +714,8 @@ def build(self):
                 else:
                     self._patch_configure()
                     self._patch_makefile_org()
-                self._make()
+                with self._make_context():
```

Comment:
> A few lines above, `env_vars` is already setting the `CC` environment variable.
> Maybe combine the sanitizing done in `_make_context` with that or refactor the `env_vars` on the top into a `_build_context` helper method?

You can't do that easily: if your environment modification affects `self._patch_configure()`, the recipe will be broken because `_patch_configure` expects `CC` variable value without quotes. Anyway this `_make_context` solves one very special problem and IMHO should not be mixed with anything else.